client/src/components/TeaFormInputs.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+
+function TeaFormInputs() {
+  const [leafType, setLeafType] = useState();
+  const [steepTime, setSteepTime] = useState();
+
+  return (
+    <div id="form-drink-specific-info">
+      <label>
+        Leaf Type:
+        <input
+          name="leafType"
+          value={leafType}
+          placeholder="Red..."
+          onChange={e => setLeafType(e.target.value)}
+        />
+      </label>
+      <label>
+        Steep Time:
+        <input
+          name="steeptime"
```

Comment:
Depending on how the post request get built, we may need to change this to steep**T**ime

Reply:
Nice catch, that's a typo on my part